tools/appliance/definitions/systemvmtemplate/configure_login.sh

```diff
@@ -23,11 +23,12 @@ function add_admin_group() {
   groupadd -f -r admin
 }
 
-function configure_cloud_user() {
+function configure_user() {
   usermod -a -G admin cloud
   mkdir -p /home/cloud/.ssh
   chmod 700 /home/cloud/.ssh
   echo "cloud:`openssl rand -base64 32`" | chpasswd
+  echo "root:password" | chpasswd
```

Comment:
Yes, this is ensure all systemvmtemplates have this default credentials (this is what the current systemvmtemplates use as well). In production env, you can set `system.vm.random.password` to true which then uses `system.vm.password` to get the password randomly set when mgmt server is initialized.